src/main/java/org/wildfly/security/sasl/otp/OTPUtil.java

```diff
@@ -330,6 +336,12 @@ public static String messageDigestAlgorithm(String algorithm) throws NoSuchAlgor
                 return MD5;
             case ALGORITHM_OTP_SHA1:
                 return SHA1;
+            case ALGORITHM_OTP_SHA_256:
+                return SHA256;
```

Comment:
This value gets used in "init-hex" and "init-word" OTP responses to specify the new OTP algorithm that should be used. This value needs to match the algorithm identifier from the algorithm name. For example, for the new algorithm name "otp-sha256", the algorithm identifier is "sha256". So the value returned here should be "sha256" instead of "SHA-256".